docs/release-info/release-notes.md

```diff
@@ -57,6 +57,12 @@ For tips about how to write a good release note, see [Release notes](https://git
 
 This section contains important information about new and existing features.
 
+### Breaking change
+
+Druid 29 has a breaking change for EXPLAIN queries.
+In the attribute field returned as part of the result for an explain query, the value of the key `targetDataSource` from a string to a JSON object.
```

Comment:
Could you add more details like it impacts only MSQ queries  and give an example of the new json object.

Reply:
This has been added on top of the release notes. Which kind of seem bad. This should be present in the upgrade section which is there near the end of the file.

Reply:
Moved to the upgrade section